solidity.hcl

```diff
@@ -0,0 +1,36 @@
+description = "Solidity is an object-oriented, high-level language for implementing smart contracts."
+homepage = "https://soliditylang.org/"
+binaries = ["solc"]
+test = "solc --version"
+
+vars = {
+    "os_": "${os}",
+    "baseurl": "https://github.com/ethereum/solidity/releases/download",
+}
+
+platform "darwin" {
```

Comment:
Just a suggestion, but I think this is probably a bit over-normalised, and loses clarity. A similar package to this one is [bazel](https://github.com/cashapp/hermit-packages/blob/master/bazel.hcl). Up to you though.